blackbird_python/blackbird/auxiliary.py

```diff
@@ -169,7 +174,13 @@ def _expression(expr):
         if expr.TIMES():
             return np.prod([_expression(a), _expression(b)], axis=0)
         if expr.DIVIDE():
-            return np.prod([_expression(a), np.power(_expression(b), -1)], axis=0)
+            a = _expression(a)
+            b = _expression(b)
+
+            if isinstance(b, int):
+                b = float(b)
```

Comment:
This was out of scope of the PR, oops. Noticed that numpy would complain if `b` was an integer, and the power was -1.

Reply:
Ok, can you make issue so we remember to fix later?

Reply:
:+1: